browser/search/ntp_utils.cc

```diff
@@ -26,14 +26,11 @@ void ClearNewTabPageProfilePrefs(Profile* profile) {
 }
 
 const char* const kWidgetPrefNames[] = {
-  kNewTabPageShowRewards,
-  kNewTabPageShowTogether,
-  kNewTabPageShowBinance,
+    kNewTabPageShowRewards, kNewTabPageShowBraveTalk, kNewTabPageShowBinance,
 #if BUILDFLAG(CRYPTO_DOT_COM_ENABLED)
-  kCryptoDotComNewTabPageShowCryptoDotCom,
+    kCryptoDotComNewTabPageShowCryptoDotCom,
 #endif
-  kNewTabPageShowGemini
-};
+    kNewTabPageShowGemini};
```

Comment:
you can add a comma here so it can be formatted a little better: ```kNewTabPageShowGemini,```

Reply:
Fixed 😄